functions.js

```diff
@@ -0,0 +1,106 @@
+const {dbConnection} = require("./db");
+
+const getAllStations = async (req, res) => {
+    const db = await dbConnection
+    try {
+        const stations = await db.query('SELECT DISTINCT `code`, `name` FROM stations Order By `name` ASC')
+        res.status(200).json({message: "Successfully Retrieved Stations.", data: stations})
+    } catch (e) {
+        res.status(500).json({message: "Unexpected Error", data: []})
+    }
+}
+
+const getJourneys = async (req, res) => {
+    try {
+        // Get Query Params & Check they're populated. Throw Bad Request.
+        const {origin: originSelection, destination: destinationSelection} = req.query
```

Comment:
I'm not saying we get rid of the comments, as I think they are very useful for us when using the code, is that a viable reason for code review?  I would say yes.

Reply:
decided to keep them